docs/pages/distribution/app-size.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Understanding app size
+description: Learn about how to determine what the actual size of your app will be when distributed to users, and how to get insights into your app size.
+---
+
+import { DocsLogo } from '@expo/styleguide';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
+
+A common concern for developers is how much space their app takes up on the app store. This guide will help you understand what different build artifacts are for, figure out the actual size of your app will be when distributed to users, and learn how to get insights into and optimize your app size.
+
+## "Why is my app so big?"
+
+**It probably isn't, actually!** When examining the resulting artifact of a release build for an app, it's common for developers who are unfamiliar with native Android and iOS development to be surprised by the file size — it's usually much larger than they would expect for an app of their scale if they were to download it from an app store. The reason is that **this is not actually the size of your app as it will be distributed on the app store!** When people talk about app sizes, what is relevant is the size of the app that they will download to their device, not the size that will be uploaded to app stores or shared in development and testing.
+
+There are various types of build artifacts that serve different purposes, and they are almost all larger than what users will see when they download your app from a store. This is because these builds are not optimized to target specific devices like they would when downloading from a store, but rather they typically include all of the code and resources that your app needs to run on a wide range of devices.
+
+## iOS apps
+
+The download size on the App Store of a minimal React Native app is just under 4 MB.
```

Comment:
```suggestion
The download size on the App Store of a minimal React Native app (created using the blank template) is just under 4 MB.
```
Do we link out to X from the docs? If so, maybe worth linking out to that tweet thread.

Reply:
hm yeah i suppose we could do that!